### DISCUSSION TITLE
Deprecated packages are restored to active

### DISCUSSION THREAD

**Discussion by JaneyDing2023** [community/community#142488] [npm]
Hey folks!

We accidentally set all versions of the @heartlandone/vega package to deprecated. We hope to change all versions of the entire package to active. Since setting it to deprecated is an irreversible process, I cannot set it to active. However, this package is extremely important to our company and is used by customers. If customers see the deprecated status, they will think that we will no longer update this package.
 
It is our blessing to use npm, which is very convenient and easy to use. Please support my request when you have time. I am very grateful for you.

<img width="1428" alt="image" src="https://github.com/user-attachments/assets/85a6a9c1-390d-4803-83da-d3a5e8c7820e">

**Comment by JaneyDing2023** [comment]
I had used the above command to resolve the problem. 
<img width="1641" alt="image" src="https://github.com/user-attachments/assets/efaa36c8-feab-496c-b2d7-8ff3a65926e1">

**Comment by monishcm** [comment]
Please let us know if the solution worked. If not, I request you to raise a support ticket.
